apps/gateway-ui/src/components/DepositCard.tsx

```diff
@@ -103,22 +106,37 @@ export const DepositCard = React.memo(function DepositCard({
           )}
         </Box>
       </Stack>
-      <Flex gap='4px'>
-        <Link
-          fontSize='md'
-          color={theme.colors.blue[600]}
-          fontFamily={theme.fonts.body}
-          _hover={{ textDecoration: 'underline' }}
-          transition={`text-decoration 1s ease-in-out`}
-          href={`https://mempool.space/address/${address}`}
-          target='_blank'
-          rel='noreferrer'
-          w='fit-content'
-        >
-          {t('deposit-card.mempool_deposit_link_text')}
-        </Link>
-        <LinkIcon color={theme.colors.blue[600]} />
-      </Flex>
+      {network && network !== Network.Regtest && (
```

Comment:
Backwards compatibility could be maintained with always rendering this, and `default`ing to the mainnet link in the switch statement.